IPython/lib/latextools.py

```diff
@@ -15,7 +14,12 @@
 from traitlets.config import get_config
 from traitlets.config.configurable import SingletonConfigurable
 from traitlets import List, Bool, Unicode
-from IPython.utils.py3compat import cast_unicode, cast_unicode_py2 as u
+from IPython.utils.py3compat import cast_unicode, cast_unicode_py2 as u, PY3
+
+if PY3:
```

Comment:
Rather than checking version, we should check directly for what we are after: `try: import bytes except: import string as bytes`